dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/manager/internal/GradleProjectInspector.java

```diff
@@ -0,0 +1,251 @@
+        if (build == null)
+            return null;
+
+        File outputDirectory = gradleProject.getBuildDirectory();
+        // TODO: If the build folder doesn't exist, do we have to build it ourselves?
+        if (outputDirectory == null)
+            return null;
+
+        IPath outputPath = new Path(outputDirectory.toString());
+
+        // The liberty-gradle-plugin config file
+        IPath libertyPluginConfigPath = outputPath.append(LibertyGradleConstants.LIBERTY_PLUGIN_CONFIG_XML);
+
+        return libertyPluginConfigPath.toFile();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public LibertyBuildPluginConfiguration populateConfiguration(File configFile, IProgressMonitor monitor) throws IOException {
+        if (monitor != null && monitor.isCanceled() || configFile == null || !configFile.exists())
+            return null;
+        Trace.trace(Trace.INFO, "Reading configFile from" + configFile.getAbsolutePath());
+        LibertyBuildPluginXMLConfigurationReader reader = new LibertyBuildPluginXMLConfigurationReader();
+        return reader.load(configFile.toURI());
+    }
+    
+    // TODO: This is where we can attach the progress listener and maintain the project connection.
+    private GradleProject getGradleProject(IProgressMonitor monitor)  {
+    		GradleConnector gradleConnector = GradleConnector.newConnector();
+    		System.out.println(project.getLocation().toString());
+    		ProjectConnection connection = gradleConnector.forProjectDirectory(new File(project.getLocation().toString())).connect();
+    		if (connection != null) {
+    			BuildEnvironment buildEnvironment = connection.model(BuildEnvironment.class).get();
+    			ModelBuilder<EclipseProject> model = connection.model(EclipseProject.class);
+    			EclipseProject eclipseProject = model.get();
+    			GradleProject gradleProject = eclipseProject.getGradleProject();
+    			connection.close();
+    			return gradleProject;
+    		}
+    		return null;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean isSupportedModule(IModule module) {
+        if (module == null) {
+            Trace.trace(Trace.INFO, "The gradle project " + project.getName() + " does not map to a supported module type.");
+            return false;
+        }
+
+        String id = module.getModuleType().getId();
+        if ("jst.web".equals(id) || "jst.ear".equals(id)) {
+            return true;
+        }
+
+        Trace.trace(Trace.INFO,
+                    "The gradle project " + project.getName() + " is detected as a module but does not specify a supported packaging type (eg. war packaging type).");
+        return false;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public IModule[] getProjectModules() {
+        IModule[] modules = new IModule[0];
+
+        LibertyGradleConstants.ProjectType projectType = getProjectType();
+        try {
+//            switch (projectType) {
```

Comment:
Remove the commented out code since we won't be supporting Liberty Assembly for Gradle